pwndbg/color/__init__.py

```diff
@@ -45,75 +45,75 @@
 none = str
 
 
-def normal(x: str) -> str:
+def normal(x: str, bold=False) -> str:
```

Comment:
It doesn't look like we're passing the `bold` argument to `colorize`?

Also, it's probably better to use `*args` to pass it up to `colorize`. If you can figure out how to type it with `ParamSpec` (there are some other examples in the codebase if you grep for it) that would be great, but if you find it tricky just use `Any`.

Reply:
@fidgetingbits ping :)

Reply:
Makes sense. To be clear, you are suggesting I change the order of colorize() args so that the color is first? That way every color wrapper can just specify the color and then pass all *args and **kwargs, so if anything gets added it's easier? Something like:

```python
def white(*args, **kwargs) -> str:
    return colorize(WHITE, args, kwargs)
```

Reply:
Yup, I think that's the right way to go. Technically you don't need to change the argument order if you directly pass `x`, `color`, and then only `bold` comes from `*args` or `**kwargs`, but I don't see a reason not to include `x` into `*args` or `**kwargs` as well. You'll need to change a few invocations of `colorize` outside of this file though if you do it like this.